lib/exercism/submission.rb

```diff
@@ -103,6 +103,7 @@ def like!(user)
   def unlike!(user)
     likes.where(user_id: user.id).destroy_all
     self.is_liked = liked_by.length > 0
+    notifications.unread.where(action: 'like', actor_id: user.id).destroy_all
```

Comment:
if `like!` doesn't know about notifications, `unlike!` should not know about notifications.
This suggests to me that this is the wrong place for this code.